Remove the /var/tmp/vi.recover directory from the package.

The /var/tmp/vi.recover directory is not required for operation. If
desired for an installation, the included vi.recover script can be
used to create it.

The relationship between APK packages and files/directories in /var/tmp is
clouded.  Avoiding the sitation seems the best approach given the above.
https://gitlab.alpinelinux.org/alpine/aports/-/issues/9364

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -584,12 +584,2 @@
 ifndef DEBUG
-	-@$(PRINTF) "\r\t%s\t%42s\n" "mkdir:" "/var/tmp/vi.recover"
-endif # DEBUG
-	@$(VERBOSE); $(TEST) -d "/var/tmp/vi.recover" ||  \
-          {                                               \
-            $(MKDIR) "/var/tmp/vi.recover" &&             \
-            $(TEST) -d "/var/tmp/vi.recover" &&           \
-            $(CHOWN) "$(IUSGR)" "/var/tmp/vi.recover" &&  \
-            $(CHMOD) "1777" "/var/tmp/vi.recover";        \
-          }
-ifndef DEBUG
 	-@$(PRINTF) "\r\t%s\t%42s\n" "mkdir:" "$(PREFIX)/bin"
